exp-player/addon/components/exp-video-physics.js

```diff
@@ -0,0 +1,172 @@
+        }
+    },
+    actions: {
+        playNext: function() {
+            window.clearTimeout(this.get('timeoutID'));
+            if (this.get('doingIntro')) { // moving to test video
+                this.set('doingIntro', false);
+                $('#player-video')
+                    .attr('loop', true)
+                    .addClass('test')
+                    .removeClass('intro attn');
+            } else {
+                this.get('next')(); // moving to intro video
+                $('#player-video')
+                    .removeClass('attn test')
+                    .addClass('intro')
+                    .attr('loop', false);
+            }
+        },
+        startVideo: function () {
+            if (!this.get('doingIntro') && !this.get('doingAttn')) {
+                var emberObj = this;
+                var t = window.setTimeout(function(emb) {
+                    emb.send('playNext');
+                    }, emberObj.get('testLength')*1000, emberObj);
+                this.set('timeoutID', t);
+            }
+        },
+        pause: function () {
+            var movie = $('#player-video')[0];
+            var emberObj = this;
+            window.clearTimeout(this.get('timeoutID'));
+            if (!emberObj.get('doingAttn') || !this.checkFullscreen()) { // pausing one of the videos
+                emberObj.set('doingAttn', true);
+                $('#player-video')
+                    .addClass('attn')
+                    .removeClass('intro test')
+                    .attr('loop', true);
+            } else { // returning to the videos
+                 // if doing intro, just return, no change necessary.
+                 // but if we were on a test video...
+                if (!emberObj.get('doingIntro')) {
+                    // if it was already the alternate, just move on.
+                    if (emberObj.get('useAlternate')) {
+                        emberObj.send('next');
+                    } else {
+                    // if we still have the alternate to use, start at intro
+                        emberObj.set('useAlternate', true);
+                        emberObj.set('doingIntro', true);
+                    }
+                }
+
+                $('#player-video').removeClass('attn test');
+                $('#player-video').addClass('intro');
+                emberObj.set('doingAttn', false);
+                $('#player-video').attr('loop', false);
+            }
+        }
+    },
+
+    didRender() { // set up event handler for pausing
+        this._super(...arguments);
+        var emberObj = this;
+        $(document).on("keypress", function (e) {
+            if (emberObj.checkFullscreen()) {
+                if (e.which == 32) { // space
+                    emberObj.send('pause');
+                    // This isn't actually a function but somehow triggering the error works anyway so I'll take it for now...
+                    //emberObj.get('refresh')();
```

Comment:
This was part of some debugging to figure out why the handler was consistently only being called the second time the event happened. The intended behavior was probably to try forcing re-rendering, maybe in case pause was being called but we didn't see the changes? And it works--consistently, on both Firefox and Chrome. (The same thing "fixes" the double-click bug in preview...) 

But of course this isn't actually a valid function call. It also works with a made-up function name. So somehow the error is enough to fix this "double-click" (/"double-press") bug.  Possibly it's just some delay that's being introduced between key events?